functions-and-operators/expressions-pushed-down.md

```diff
@@ -42,29 +42,29 @@ Field description:
 
 + `name`: the name of the function that is prohibited from being pushed down.
 + `store_type`: specifies to which storage engine the function is prohibited from being pushed down. Currently, TiDB supports the three storage engines: `tikv`, `tidb`, and `tiflash`. `store_type` is case-insensitive. If a function is prohibited from being pushed down to multiple storage engines, use a comma to separate each engine.
-+ `reason` : The reason why the function is blacklisted.
++ `reason` : The reason why the function is blocklisted.
 
-### Add to the blacklist
+### Add to the blocklist
 
-To add one or more functions or operators to the blacklist, perform the following steps:
+To add one or more functions or operators to the blocklist, perform the following steps:
 
 1. Insert the function or operator name and the collection of storage types to be prohibited from the function pushdown to `mysql.expr_pushdown_blacklist`.
 
 2. Execute the `admin reload expr_pushdown_blacklist;` command.
```

Comment:
It seems that we need to modify words in the two steps first, to keep consistency and avoid confusion.